src/Template/Pages/home.ctp

```diff
@@ -42,71 +42,71 @@ $cakeDescription = 'CakePHP: the rapid development php framework';
     <header>
         <div class="header-image">
             <?= $this->Html->image('http://cakephp.org/img/cake-logo.png') ?>
-            <h1>Get the Ovens Ready</h1>
+            <h1><?= __d('cake', 'Get the Ovens Ready') ?></h1>
         </div>
     </header>
     <div id="content">
         <div class="row">
             <div class="row">
                 <div class="columns large-12 ctp-warning checks">
-                    <p>Please be aware that this page will not be shown if you turn off debug mode unless you disable the NotFoundException in src/Template/Pages/home.ctp.</p>
+                    <p><?= __d('cake', 'Please be aware that this page will not be shown if you turn off debug mode unless you disable the NotFoundException in src/Template/Pages/home.ctp.') ?></p>
```

Comment:
I do agree that errors and warnings should reside in its own domain. There is no problem then as one can omit translation? Putting them in the cake domain is not good.